Record_Force_Sensor.m

```diff
@@ -1,99 +0,0 @@
-%Written by Kenneth Cheung on 2/27/21

```

Comment:
This doesn't have anything to do with ground station, right? Make sure whatever this is has a known location

Reply:
Got moved to ground_testing/